.github/workflows/trigger-gitlab.yml

```diff
@@ -1,35 +1,15 @@
 # inspired by rhinstaller/anaconda
 
 name: Trigger GitLab CI
-on: [push, pull_request_target]
 
-jobs:
-  pr-info:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Query author repository permissions
-        uses: octokit/request-action@v2.x
-        id: user_permission
-        with:
-          route: GET /repos/${{ github.repository }}/collaborators/${{ github.event.sender.login }}/permission
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      # restrict running of tests to users with admin or write permission for the repository
-      # see https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-repository-permissions-for-a-user
-      # store output if user is allowed in allowed_user job output so it has to be checked in downstream job
-      - name: Check if user does have correct permissions
-        if: contains('admin write', fromJson(steps.user_permission.outputs.data).permission)
-        id: check_user_perm
-        run: |
-          echo "User '${{ github.event.sender.login }}' has permission '${{ fromJson(steps.user_permission.outputs.data).permission }}' allowed values: 'admin', 'write'"
-          echo "::set-output name=allowed_user::true"
-    outputs:
-      allowed_user: ${{ steps.check_user_perm.outputs.allowed_user }}
+on:
+  workflow_run:
+    workflows: ["Tests"]
+    types: [completed]
 
+jobs:
   trigger-gitlab:
-    needs: pr-info
-    if: needs.pr-info.outputs.allowed_user == 'true'
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
Is this is even needed? :thinking:  Otherwise, this is looking good.

Reply:
I think the possible conditions for the event are `requested` and `completed`, but the latter does not imply successfully completed, hence the check.

Reply:
Yeah... I'm still not 100% sure if it's really needed but it shouldn't hurt anything.